acceptance/tests/windows/prevent_password_changes.rb

```diff
@@ -0,0 +1,104 @@
+require 'date'
+
+def random_username
+  "pl#{rand(999999).to_i}"
+end
+
+def change_password_manifest(username)
+  return <<-MANIFEST
+    user { '#{username}':
+      ensure   => 'present',
+      password => 'Password-#{rand(999999).to_i}'
+    }
+  MANIFEST
+end
+
+def assert_password_matches_on(host, username, password)
+  script = <<-PS1
+  Add-Type -AssemblyName System.DirectoryServices.AccountManagement
+  $ctx = New-Object System.DirectoryServices.AccountManagement.PrincipalContext("machine", $env:COMPUTERNAME)
+  $ctx.ValidateCredentials("#{username}", "#{password}")
+  PS1
+  execute_powershell_script_on(host, script) do |result|
+    assert_match(/True/, result.stdout, "Expected the the user's password not to be updated, but it was")
+  end
+end
+
+OLD_PASSWORD='0ldP@ssword'
+
+hosts.each do |host|
+  test_name 'PUP-6569 Passwords are not reset for disabled accounts' do
```

Comment:
Do other tests use multiple `test_name` blocks in a single file?

Reply:
I don't know - these are some of my first beaker tests. I'll look through the rest and try to find a more appropriate pattern.